demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -969,6 +969,31 @@ def test_invalid_integration_path(self, integration, mocker):
 
         assert not validator.is_valid_integration_file_path()
 
+    @pytest.mark.parametrize('file_name', ['IntNameTest.py', 'IntNameTest_test.py'])
+    def test_invalid_py_file_names(self, repo, file_name):
+        """
+        Given
+            - An integration with invalid python file path.
```

Comment:
There are two different cases here, let's address them separately as they belong to two different flows.

Reply:
Do you mean split the `integration.py` and the `_test.py`?
If so why?

Reply:
If I understand this test correctly - you are validating onc an integration py file and once a unit test py file, to make the test clearer it is better to address each of them separately in the docstring such that the next dev that looks at the logic understands exactly the two different use cases being checked here.

Reply:
As you wish